examples/simple/main.cc

```diff
@@ -11,19 +11,22 @@ namespace
 {
 void initTracer()
 {
-  auto exporter  = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  auto exporter = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  // Specify the Span Processor: SimpleSpanProcessor forwards spans directly to the exporter
   auto processor = std::shared_ptr<sdktrace::SpanProcessor>(
       new sdktrace::SimpleSpanProcessor(std::move(exporter)));
   auto provider = nostd::shared_ptr<trace::TracerProvider>(new sdktrace::TracerProvider(processor));
+  // Set the global trace provider
   trace::Provider::SetTracerProvider(provider);
 }
 }  // namespace
 
 int main()
 {
-  // Removing this line will leave OT initialized with the default noop
+  // Removing this line will leave OTel SDK initialized with the default noop
```

Comment:
This has brought an interesting topic, neither the term `OT` nor `OTel` has been officially blessed https://github.com/open-telemetry/opentelemetry-specification/search?q=OTel&unscoped_q=OTel.

Reply:
I'll use "OpenTelemetry" to avoid confusion in that case.